.github/workflows/build-scan-push.yaml

```diff
@@ -37,7 +37,6 @@ jobs:
         run: go test -v ./... --count=1
 
   build-image:
-    needs: build-tests
```

Comment:
@gtardif The reason for having the `needs` is to avoid building and pushing the image in case the Go tests fail.

Reply:
i'll keep it then